devsetup/Makefile

```diff
@@ -377,25 +378,39 @@ edpm_baremetal_compute: export OPERATOR_NAME=openstack
 edpm_baremetal_compute: export NAMESPACE=${BMH_NAMESPACE}
 edpm_baremetal_compute: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
 edpm_baremetal_compute: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
+edpm_baremetal_compute: export BMAAS_INSTANCE_NAME_SUFFIX=${BM_INSTANCE_NAME_SUFFIX}
 edpm_baremetal_compute: export BMAAS_NODE_COUNT=${BM_NODE_COUNT}
 edpm_baremetal_compute: export NODE_COUNT=${BM_NODE_COUNT}
 edpm_baremetal_compute: export DEPLOY_DIR=../out/bmaas
 edpm_baremetal_compute: ## Create virtual baremetal for the dataplane
 	$(eval $(call vars))
 	make bmaas_virtual_bms
 	make bmaas_sushy_emulator_wait
-	bash scripts/edpm-compute-baremetal.sh --create
+	if [ "${NODE_COUNT}" = "1" ]; then \
+		bash scripts/edpm-compute-baremetal.sh --create ; \
+	else \
+		for INDEX in $(shell seq 0 $$((${NODE_COUNT} -1))) ; do \
+			export BMAAS_INSTANCE_NAME_SUFFIX=$$INDEX ; \
+			bash scripts/edpm-compute-baremetal.sh --create ; \
+		done \
+	fi
 
 edpm_baremetal_compute_cleanup: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
 edpm_baremetal_compute_cleanup: export NAMESPACE=${BMH_NAMESPACE}
 edpm_baremetal_compute_cleanup: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
+edpm_baremetal_compute_cleanup: export BMAAS_INSTANCE_NAME_SUFFIX=${BM_INSTANCE_NAME_SUFFIX}
 edpm_baremetal_compute_cleanup: export BMAAS_NODE_COUNT=${BM_NODE_COUNT}
 edpm_baremetal_compute_cleanup: export NODE_COUNT=${BM_NODE_COUNT}
 edpm_baremetal_compute_cleanup:  ## Cleanup dataplane with BMAAS
 	$(eval $(call vars))
-	scripts/edpm-compute-baremetal.sh --cleanup
-	pushd .. && rm -Rf out/edpm || true && popd
-	make bmaas_sushy_emulator_cleanup || true
+	if [ "${NODE_COUNT}" = "1" ]; then \
```

Comment:
As we're changing the name of the vms and bmhs, cleanup won't work with things created with earlier version of the script. They have to be manually deleted.

Reply:
I think that would still be possible with using the env vars to set the names. However, I don't think we need to maintain backwards compatibility here anyway.